ui/plugin-system/src/runtime/DataQueriesProvider/DataQueriesProvider.tsx

```diff
@@ -12,16 +12,17 @@
 // limitations under the License.
 
```

Comment:
Is it worth adding a couple simple unit tests for some of the logic changes in this file?

Reply:
yep done!